files/magneticmediajs/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Arnaud Leyder"
+github = "https://github.com/arnaudleyder"
```

Comment:
Yes that is correct, I will keep it in mind for a next time. I have committed changes to https://github.com/arnaudleyder/jsdelivr do I need to send a new pull request?